projects/spectator/test/todos-data.service.spec.ts

```diff
@@ -35,6 +35,21 @@ describe('HttpClient testing', () => {
     spectatorHttp.expectOne('two', HttpMethod.GET);
   });
 
+  it('should test concurrent requests', () => {
+    const spectatorHttp = http();
+
+    spectatorHttp.service.concurrentRequests().subscribe();
+    const reqs = spectatorHttp.expectConcurrent([
```

Comment:
It can be cleaner: 
`spectatorHttp.expectConcurrent(['one', 'two', 'three'].map(url => 
    ({ url, method:HttpMethod.POST})))`